.circleci/continue_config.yml

```diff
@@ -63,6 +63,29 @@ jobs:
           name: Run doc tests
           command: make doc-test
 
+  style_and_unit_tests_pydantic_v2:
+    docker:
+      - image: cimg/python:3.10
```

Comment:
Do you feel like 3.10 is sufficient? Is there an CircleCI impact if we test on the latest and oldest supported python versions too?

Reply:
Yeah, it'd take longer and cost more credits. I'm not sure I see the value though. Python versions and Pydantic versions are orthogonal to me.

Reply:
90% of the time when I have a CircleCI issue, its either Python 3.7 or 3.11 related. If our pydantic related code has a Python version issue that only manifests in Pydantic V2, we won't know. It'll be a gap in our confidence that the code works imho.

I was under the impression CircleCI was free since we are open source. How does it actually work?

Reply:
3.10 is fine